### PR TITLE
Toast.test.tsx

### PR DESCRIPTION
**Pull request by Hershey2776**
### Pull Request Checklist: 

- [ ] Please make sure all jobs pass before requesting a review. 
- [ ] Assign yourself to the PR
- [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
- [ ] Label it as "needs-review"
- [ ] Have you lint your code locally before submission?
- [ ] Did you write tests where appropiate?

### CODE REVIEW

File: CommonUI/src/Components/Toast/Toast.tsx [hunk 1]
```diff
@@ -23,28 +23,34 @@ const Component: FunctionComponent<ComponentProps> = (
...
-    if (props.type === ToastType.WARNING) {
-        typeCssClass = 'text-warning';
+    if (props.type === ToastType.INFO) {
+        typeCssClass = 'text-info';
     }
 
     if (show) {
         return (
             <div
+                id="main"
```
Comment (simlarsen) [inline CommonUI/src/Components/Toast/Toast.tsx:45]: change to toast-main

File: CommonUI/src/Components/Toast/Toast.tsx [hunk 2]
```diff
@@ -23,28 +23,34 @@ const Component: FunctionComponent<ComponentProps> = (
...
                 className="position-fixed top-0 end-0 p-3"
                 style={{ zIndex: '1005' }}
             >
                 <div className="toast fade show" role="alert">
                     <div className="toast-header">
                         {props.type && (
                             <div
+                                id="status"
```
Comment (simlarsen) [inline CommonUI/src/Components/Toast/Toast.tsx:53]: change to toast-status

File: CommonUI/src/Components/Toast/Toast.tsx [hunk 3]
```diff
@@ -53,14 +59,17 @@ const Component: FunctionComponent<ComponentProps> = (
                                 </span>
                             </div>
                         )}
-                        <strong className="me-auto ms-2">{props.title}</strong>
+                        <strong id="strong" className="me-auto ms-2">
```
Comment (simlarsen) [inline CommonUI/src/Components/Toast/Toast.tsx:62]: id change here as well

File: CommonUI/src/Components/Toast/Toast.tsx [hunk 4]
```diff
@@ -53,14 +59,17 @@ const Component: FunctionComponent<ComponentProps> = (
...
+                            {props.title}
+                        </strong>
                         {props.createdAt && (
                             <small>
                                 {OneUptimeDate.fromNow(props.createdAt)}
                             </small>
                         )}
 
                         <button
+                            id="button"
```
Comment (simlarsen) [inline CommonUI/src/Components/Toast/Toast.tsx:72]: id change here as well,.

**Review by simlarsen — CHANGES_REQUESTED** [review]

File: CommonUI/tsconfig.json [hunk 5]
```diff
@@ -95,7 +95,7 @@
         "noFallthroughCasesInSwitch": true /* Enable error reporting for fallthrough cases in switch statements. */,
         "noUncheckedIndexedAccess": true /* Include 'undefined' in index signature results */,
         "noImplicitOverride": true /* Ensure overriding members in derived classes are marked with an override modifier. */,
-        "noPropertyAccessFromIndexSignature": true /* Enforces using indexed accessors for keys declared using an indexed type */,
+        "noPropertyAccessFromIndexSignature": false /* Enforces using indexed accessors for keys declared using an indexed type */,
```
Comment (simlarsen) [inline CommonUI/tsconfig.json:98]: Why change this?

**Review by simlarsen — CHANGES_REQUESTED** [review]

File: CommonUI/tsconfig.json (hunk 5 above)
Comment (Hershey2776) [reply CommonUI/tsconfig.json:98]: I was getting a error "Property 'xyz' comes from an index signature". This fixes the error

**Pull request merged**